src/main/scala/au/com/agiledigital/toolform/model/Component.scala

```diff
@@ -24,9 +25,9 @@ final case class Component(idOverride: Option[String],
                            settings: Option[Config],
                            componentGroup: Option[ComponentGroup],
                            environment: Map[String, String] = Map(),
-                           exposedPorts: List[String] = List())
-    extends ProjectElement
-    with Service {
+                           exposedPorts: List[String] = List(),
```

Comment:
These should be `PortMapping` instead of `String` now.

Reply:
Fixed in c2dd744